markets/models.py

```diff
@@ -84,14 +84,10 @@ def __str__(self) -> str:
         return f"{self.title} ({self.slug})"
 
     def get_absolute_url(self) -> str:
-        if self.has_ended:
+        if self.has_ended or not self.has_started:
```

Comment:
IMO flipping the if statement `if self.has_started and not self.has_ended` is clearer